src/pages/resultsView/enrichments/EnrichmentsUtil.tsx

```diff
@@ -436,6 +635,19 @@ export function getFilteredData(
     });
 }
 
+export function getFilteredCategoricalData(
+    data: GenericAssayCategoricalEnrichmentRow[],
+    filterFunction: (value: string) => boolean
+): any[] {
```

Comment:
If there is a specific type to use, maybe we can just use it: `GenericAssayCategoricalEnrichmentRow`. I know it's probably existing somewhere else, just wish to mention that to you.

Reply:
Okay, done.